resources/user-guides/README.md

```diff
@@ -1,17 +1,48 @@
+The following tutorials guide you through the implementation of the Platform Mobile SDK in a sample retail app called Luma. The Luma app has functionality that lets you build a realistic implementation. The lessons are designed for iOS and written in Swift, but many of the concepts also apply to Android.
 
-* [Implementing the Experience Cloud in Mobile iOS Swift Applications](https://experienceleague.adobe.com/docs/launch-learn/implementing-in-mobile-ios-swift-apps-with-launch/index.html)
-* [Implementing the Experience Cloud in Mobile Android Applications](https://experienceleague.adobe.com/docs/launch-learn/implementing-in-mobile-android-apps-with-launch/index.html)
+* [Implement Platform Mobile SDKs in mobile applications](https://experienceleague.adobe.com/docs/platform-learn/implement-mobile-sdk/overview.html)
+
+## Getting started with Adobe Experience Platform
+
+This tutorial illustrates how you may send Experience Data Model (XDM) commerce events to Adobe Experience Platform via Experience Edge using the AEP Edge extension in a sample application, provided to you in iOS (Swift) and Android.
+
+* [Getting started with Adobe Experience Platform](https://aep-sdks.gitbook.io/docs/resources/user-guides/getting-started-with-adobe-experience-platform)
 
 ## Tracking Beacons in Mobile Android and iOS applications
 
-Sending beacon data to Adobe Analytics for reporting can be an effective to understand how your audience can interact with physical landmarks. You can manually send beacon tracking data to your Analytics server and create rules based on a user's proximity to your beacons. This solution relies on the Profile extension to generate beacon-related rules.
+Sending beacon data to Adobe Analytics for reporting can be an effective to understand how your audience can interact with physical landmarks. 
 
-For more information, see the following link:
+This tutorial illustrates how to manually send beacon tracking data to your Analytics server and create rules based on a user's proximity to your beacons. This solution relies on the Profile extension to generate beacon-related rules.
 
 * [Implementing beacon tracking for Adobe Analytics in Mobile Android and iOS applications](track-beacon.md)
 
+## Attach or modify data in SDK events
+
+These tutorials illustrate how to enable advanced use-cases with Attach Data and Modify Data actions.
+
+* [Attach data to SDK events](attach-data.md)
+* [Modify data to SDK events](modify-data.md)
+
+## Using data elements to define dynamic data in rules
+
+Data elements are building blocks for rules and allow you create a data dictionary of commonly used data in the Platform Mobile SDK. This collection of data elements becomes the dictionary of defined data that you can use to build rules for your application, and this dictionary is shared across Experience Platform Launch where it can be used with any extension that is added to your mobile property.
```

Comment:
Since Launch is no longer that brand name -

"...shared across your tags implementation where it can..."